aws/main.tf

```diff
@@ -0,0 +1,274 @@
+# Configure the Amazon AWS Provider
+provider "aws" {
+  access_key = "${var.aws_access_key}"
+  secret_key = "${var.aws_secret_key}"
+  region     = "${var.region}"
+}
+
+variable "aws_access_key" {
+  default     = "xxx"
+  description = "Amazon AWS Access Key"
+}
+
+variable "aws_secret_key" {
+  default     = "xxx"
+  description = "Amazon AWS Secret Key"
+}
+
+variable "prefix" {
+  default     = "yourname"
+  description = "Cluster Prefix - All resources created by Terraform have this prefix prepended to them"
+}
+
+variable "rancher_version" {
+  default     = "v2.0.6"
+  description = "Rancher Server Version"
+}
+
+variable "count_agent_all_nodes" {
+  default     = "3"
```

Comment:
we are changing this to a default of 1 for all quickstarts, and instructions need to be updated to show how to increase this